lib/services/visualSearch/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "azure-cognitiveservices-visualsearch",
+  "author": "Microsoft Corporation",
+  "description": "VisualSearchAPIClient Library with typescript type definitions for node",
+  "version": "1.0.0-preview",
```

Comment:
Make sure to verify with Kirthi that she wants this to be a preview package instead of just "1.0.0".

Reply:
Good catch--yeah, that one was called out as needing to be preview.